openpype/hosts/zbrush/addon.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import os
+from openpype.modules import OpenPypeModule, IHostAddon
+
+ZBRUSH_HOST_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_launch_script_path():
+    return os.path.join(
+        ZBRUSH_HOST_DIR,
+        "api",
+        "launch_script.py"
+    )
+
+
+class ZbrushAddon(OpenPypeModule, IHostAddon):
+    name = "zbrush"
+    host_name = "zbrush"
+
+    def initialize(self, module_settings):
+        self.enabled = True
+
+    def add_implementation_envs(self, env, _app):
+        # Remove auto screen scale factor for Qt
+        # - let 3dsmax decide it's value
+        new_zbrush_paths = [
+            os.path.join(ZBRUSH_HOST_DIR, "api", "zscripts")
+        ]
+        old_zbrush_path = env.get("ZBRUSH_PLUGIN_PATH") or ""
+        for path in old_zbrush_path.split(os.pathsep):
+            if not path:
+                continue
+
+            norm_path = os.path.normpath(path)
+            if norm_path not in new_zbrush_paths:
+                new_zbrush_paths.append(norm_path)
+
+        env["ZBRUSH_PLUGIN_PATH"] = os.pathsep.join(new_zbrush_paths)
+        defaults = {
+            "OPENPYPE_LOG_NO_COLORS": "True",
+            "WEBSOCKET_URL": "ws://localhost:6001"
+        }
+        for key, value in defaults.items():
+            if not env.get(key):
+                env[key] = value
+
+        env.pop("QT_AUTO_SCREEN_SCALE_FACTOR", None)
```

Comment:
```suggestion
        # Add AYON zscripts
        new_zbrush_paths = [
            os.path.join(ZBRUSH_HOST_DIR, "api", "zscripts")
        ]
        old_zbrush_path = env.get("ZBRUSH_PLUGIN_PATH") or ""
        for path in old_zbrush_path.split(os.pathsep):
            if not path:
                continue

            norm_path = os.path.normpath(path)
            if norm_path not in new_zbrush_paths:
                new_zbrush_paths.append(norm_path)

        env["ZBRUSH_PLUGIN_PATH"] = os.pathsep.join(new_zbrush_paths)
        
        # Set some applications defaults
        defaults = {
            "OPENPYPE_LOG_NO_COLORS": "True",
            "WEBSOCKET_URL": "ws://localhost:6001"
        }
        for key, value in defaults.items():
            if not env.get(key):
                env[key] = value

        # Remove auto screen scale factor for Qt
        env.pop("QT_AUTO_SCREEN_SCALE_FACTOR", None)
```

Make sure the comments are correct and are relevant to the code they are nearby to.
Also, the `api/zscripts` folder does not exist?
Additionally, I think the folder should not be within `api/` but in something like `deploy/` or whatever. 

In AYON these are some similar uses:
```
hosts/houdini/startup
hosts/houdini/resources
hosts/fusion/deploy
hosts/flame/startup
hosts/blender/blender_addon/startup
hosts/max/startup
hosts/maya/startup
hosts/nuke/startup
hosts/resolve/utility_scripts
hosts/substancepainter/deploy
hosts/unreal/integration
```

As you can see the naming convention used across OpenPype hosts codebase varies greatly. :) But it'd be good to at least stick to one of them and start aligning more where we can.